src/server/game/Handlers/ChatHandler.cpp

```diff
@@ -384,7 +384,7 @@ void WorldSession::HandleChatMessage(ChatMsg type, uint32 lang, std::string msg,
             if (Channel* chn = ChannelMgr::GetChannelForPlayerByNamePart(target, sender))
             {
                 sScriptMgr->OnPlayerChat(sender, type, lang, msg, chn);
-                chn->Say(sender->GetGUID(), msg.c_str(), lang);
+                chn->Say(sender->GetGUID(), msg.c_str(), lang, nullptr);
```

Comment:
So in Channel.h, for the function do: `void Say(ObjectGuid const& guid, std::string const& what, uint32 lang, std::string prefix = "") const;`, in Channel.cpp, make the function `void Channel::Say(ObjectGuid const& guid, std::string const& what, uint32 lang, std::string prefix /*= ""*/) const`, then just "" instead of nullptr?